README.md

```diff
@@ -1653,6 +1653,22 @@ Other Style Guides
     const truthyCount = array.filter(Boolean).length;
     ```
 
+  <a name="variables--space-equal-sign"></a><a name="13.7"></a>
```

Comment:
This would be a great addition to our guide as well; can you open a new PR that adds it?